### DISCUSSION TITLE
Inconsistent history display depending on entity type

### DISCUSSION THREAD

**Discussion by ve6rah** [home-assistant/frontend#16043] [Other feature requests]
When clicking an entity, you see a pop-up with its current status. For some entities the history also appears there, for others you have to click the history icon at the top.

The way the history displays is also radically different, if the history of a (non-numeric) sensor shows on the initial page without clicking, then there is no logbook component, only the coloured bar, but if you do have to click a history icon, you get both the coloured bar, and the logbook for that entity.

This display should be consistent between different types of entities, and should always include the logbook if it is available.

**Comment by ve6rah** [comment]
Examples:
- binary_sensor, switch, light, lock, cover, and person entities all show with a separate history page including logbook
- sensor entities show with the history displayed inline with the current status and no logbook.

**Reply by ildar170975** [reply]
Previously there were more logic in behaviour, after 2023.3 it became bad (nice, shiny - but useless).
